test/projections.jl

```diff
@@ -130,3 +130,62 @@ end
     end
     @test all(case_p .> 0) && all(case_d .> 0)
 end
+
+@testset "Power Cone Projections" begin
+    function det_case_pow_cone(x, α; dual=false)
+        v = dual ? -x : x
+        s = MOI.PowerCone(α)
+        if MOD._in_pow_cone(v, s)
+            return 1
+        elseif MOD._in_pow_cone(v, MOI.dual_set(s))
+            return 2
+        elseif abs(v[3]) <= 1e-8
+            return 3
+        else
+            return 4
+        end
+    end
+
+    function _test_proj_pow_cone_help(x, α, tol; dual=false)
+        cone = dual ? MOI.DualPowerCone(α) : MOI.PowerCone(α)
+        model = Model()
+        set_optimizer(model, optimizer_with_attributes(
+            SCS.Optimizer, "eps" => 1e-10, "max_iters" => 10000, "verbose" => 0))
+        @variable(model, z[1:3])
+        @variable(model, t)
+        @objective(model, Min, t)
+        @constraint(model, sum((x-z).^2) <= t)
+        @constraint(model, z in cone)
+        optimize!(model)
+        z_star = value.(z)
+        px = MOD.projection_on_set(DD, x, cone)
+        if !isapprox(px, z_star, atol=tol)
+            # error("x = $x\nα = $α\nnorm = $(norm(px - z_star))\npx=$px\ntrue=$z_star")
```

Comment:
let's remove this if not used anymore

Reply:
Would prefer to uncomment and leave in the error messages if it's okay stylistically